### PR TITLE
Fix `grpc-message` typo in call-stream.ts

### PR DESCRIPTION
**Pull request by clww**
In PR https://github.com/grpc/grpc-node/pull/2210, error handling code was added for `decodeURI(metadataMap['grpc-message'])`, but there is a typo in the `catch` block where `grpc-message` was written as `grpc-messages` with an extra "**s**".

### CODE REVIEW

File: packages/grpc-js/src/call-stream.ts
Comment (murgatroid99) [inline packages/grpc-js/src/call-stream.ts:494]:
```suggestion
        details = metadataMap['grpc-message'];
```

The `as string` type assertion is only needed because the key didn't match the one in the type check in the surrounding `if` statement, so it can be removed.

**Review by murgatroid99 — CHANGES_REQUESTED** [review]
Thank you for the correction. I can't merge this unless you sign the CLA.

**Comment by clww** [comment]
The EasyCLA status hasn't been updated after I signed (probably related to multiple email associtaed with my GH account?). Could you proceed with another PR? I can close this PR.

**Comment by murgatroid99** [comment]
Unfortunately, the change needs to be against the v1.6.x branch for me to publish it, and switching the branch target for this PR pulls in the commits from master. I will make the change myself in #2216.

**Comment by murgatroid99** [comment]
I published the updated fix in version 1.6.12

**Comment by clww** [comment]
thank you!

**Pull request closed without merging**
